### DISCUSSION TITLE
trang update tổng thời gian code của dòng tộc ông dev

### DISCUSSION THREAD

**Discussion by cutbypham** [OngDev/community#186] [Project Ideas]
- Login bằng gh
- Gh dashboard
- Start/Pause/Stop
- Rank
- Server: .net core
- App electron: html css js
- Dashboard page (plain js): html css js

**Comment by cutbypham** [comment]
- có thể tích hợp pomodoro timer
- tích hợp api của  WakaTime,... (extention trên vscode,...)

**Comment by duynguyenod** [comment]
-Login bằng gh
-Gh dashboard
-Start/Pause/Stop
-Rank

**Comment by duynguyenod** [comment]
-Server: .net core
-App electron: html css js
-Dashboard page (plain js): html css js

**Comment by datnguyennnx** [comment]
- Add task / complete task ( Private/ Public ) - "Để control task chung 1 app cho dễ anh ơi"
